homeassistant/components/hue/strings.json

```diff
@@ -44,21 +44,32 @@
       "dim_down": "Dim down",
       "dim_up": "Dim up",
       "turn_off": "Turn off",
-      "turn_on": "Turn on"
+      "turn_on": "Turn on",
+      "1": "First button",
+      "2": "Second button",
+      "3": "Third button",
+      "4": "Fourth button"
     },
     "trigger_type": {
       "remote_button_long_release": "\"{subtype}\" button released after long press",
       "remote_button_short_press": "\"{subtype}\" button pressed",
       "remote_button_short_release": "\"{subtype}\" button released",
       "remote_double_button_long_press": "Both \"{subtype}\" released after long press",
-      "remote_double_button_short_press": "Both \"{subtype}\" released"
+      "remote_double_button_short_press": "Both \"{subtype}\" released",
+
+      "initial_press": "Button \"{subtype}\" pressed initially",
+      "repeat": "Button \"{subtype}\" held down",
+      "short_release": "Button \"{subtype}\" released after short press",
+      "long_release": "Button \"{subtype}\" released after long press",
+      "double_short_release": "Both \"{subtype}\" released"
     }
   },
   "options": {
     "step": {
       "init": {
         "data": {
           "allow_hue_groups": "Allow Hue groups",
+          "allow_hue_scenes": "Allow Hue scenes",
```

Comment:
Let's not make this an option. We can make the entities disabled by default.

Group is an option because they had no unique ID in the past. Now that they have a unique ID, this option could also be deprecated.

Reply:
I totally agree, users can disable entities they do not want. 
The same counts for the option that allows unavailabe devices to report their status. This is handled much better in the V2 api and is no longer needed. Can I move that one too to v1 only ?

Reply:
deprecated all config flow options for V2 bridges